data/formatters/generic.yaml

```diff
@@ -462,6 +462,24 @@ short_message:
 - '{file_md5}'
 ---
 type: 'conditional'
+data_type: 'nids:alert:fast'
+message:
+- 'Message: {message}'
+- 'Source IP: {source_ip}'
+- 'Source Port: {source_port}'
+- 'Destination IP: {destination_ip}'
+- 'Destination Port: {destination_port}'
+- 'Protocol: {protocol}'
+- 'Classification: {classifciation}'
+- 'Rule ID: {rule_id}'
+- 'Priority: {priority}'
+short_message:
+- 'Message: {message}'
+- 'Source-IP: {source_ip}'
+- 'Destination-IP: {destination_ip}'
+- 'Rule ID: {sid}'
```

Comment:
I assume this should be rule_id

Reply:
Changed in 9412614b863062cb598cb06c6acbf84d04d82f71.